hoot-core/src/main/cpp/hoot/core/io/OsmMapReader.h

```diff
@@ -84,6 +84,24 @@ class OsmMapReader
    * @return a formats string
    */
   virtual QString supportedFormats() = 0;
+
+  /**
+   * Gets the ignore duplicates flag
+   */
+  bool getIgnoreDuplicates() { return _ignoreDuplicates; }
+
+  /**
+   * Set the ignore duplicates flag, when set to true, derived classes will ignore any element
+   * who's ID is already being used.  This is useful when elements cross bounding box boundaries
+   * and multiple bounding boxes are being merged together.
+   */
+  void setIgnoreDuplicates(bool ignore) { _ignoreDuplicates = ignore; }
```

Comment:
After looking for the best place to put this, I found that `OsmMapReader` would be the best place so that all readers could use it if so desired.